packages/react-dom/src/client/ReactDOMComponentTree.js

```diff
@@ -30,8 +31,19 @@ const randomKey = Math.random()
   .toString(36)
   .slice(2);
 const internalInstanceKey = '__reactFiber$' + randomKey;
-const internalEventHandlersKey = '__reactEvents$' + randomKey;
+const internalPropsKey = '__reactProps$' + randomKey;
```

Comment:
The last time we renamed these fields, [we posted about it](https://fb.workplace.com/groups/react.core.team/permalink/3470160429699214/) to let our few semi-supported use cases know ahead of time that it was coming.

I don't *think* anything relied on `__reactEvents` (or previously `__reactEventHandlers`) so I think this change is probably fine, but cc @gaearon to confirm.

Reply:
I looked internally and it looks good to go with this PR.

Reply:
Yeah this particular one is safe

Reply:
This wasn't a rename btw. The old name is still there below.

Reply:
Right but it's been repurposed, so...kind of the same as a rename.